src/main/java/org/springframework/retry/support/RetryTemplateBuilder.java

```diff
@@ -306,6 +307,50 @@ public RetryTemplateBuilder notRetryOn(Class<? extends Throwable> throwable) {
 		return this;
 	}
 
+	/**
+	 * Add all throwables to the while list of retryable exceptions.
+	 * <p>
+	 * Warn: touching this method drops default {@code retryOn(Exception.class)}  and you
+	 * should configure whole classifier from scratch.
+	 * <p>
+	 * You should select the way you want to configure exception classifier: white list or
+	 * black list. If you choose white list - use this method, if black - use
+	 * {@link #notRetryOn(Class)} or {@link #notRetryOn(List)}
+	 * @param throwables to be retryable (with it's subclasses)
+	 * @return this
+	 * @see BinaryExceptionClassifierBuilder#retryOn
+	 * @see BinaryExceptionClassifier
+	 * @since 1.3.2
```

Comment:
The `@since` has to go just directly after `@return` before `@see`.

Our code style rule is like this:
```
<property name="tagOrder" value="@param, @return, @throws, @since, @deprecated, @see"/>
```